apps/settings/src/views/Users.vue

```diff
@@ -79,19 +79,16 @@
 					:count="group.count" />
 			</template>
 			<template #footer>
-				<NcAppNavigationSettings>
+				<NcAppNavigationSettings exclude-click-outside-selectors=".vs__dropdown-menu">
 					<div>
-						<p>{{ t('settings', 'Default quota:') }}</p>
-						<NcMultiselect :value="defaultQuota"
+						<label for="default-quota-multiselect">{{ t('settings', 'Default quota:') }}</label>
+						<NcSelect v-model="defaultQuota"
+							input-id="default-quota-multiselect"
 							:options="quotaOptions"
-							tag-placeholder="create"
 							:placeholder="t('settings', 'Select default quota')"
-							label="label"
-							track-by="id"
-							:allow-empty="false"
-							:taggable="true"
-							@tag="validateQuota"
-							@input="setDefaultQuota" />
+							:close-on-select="false"
+							@open="validateQuota"
```

Comment:
```suggestion
							@option:created="validateQuota"
```
Seems to match https://vue-select.org/api/events.html#option-created?

Reply:
And needed the quota arg https://github.com/nextcloud/server/blob/784059fb3800636521d1e8af152bdac7b7f716ae/apps/settings/src/views/Users.vue#L382

Reply:
And needed for the quota arg https://github.com/nextcloud/server/blob/784059fb3800636521d1e8af152bdac7b7f716ae/apps/settings/src/views/Users.vue#L382

Reply:
have i resolved it?

Reply:
Don't think it works currently as the `:taggable="true"` is needed as described in https://vue-select.org/api/events.html#option-created

Also you'll have to make sure the quota passed to `validateQuota()` is a string and not an object